### PR TITLE
get_spend_tx might return null

### PR DESCRIPTION
**Pull request by danielabrozzoni**
Needs https://github.com/revault/practical-revault/pull/99 and https://github.com/revault/revault_net/pull/68

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ We test message processing against Postgre. Therefore the tests need a running i
 of Postgre to connect to, `postgresql://revault:revault@localhost/coordinator_db`. The easiest
 way to set one up is by using docker:
 ```
-docker run --rm -d -p 5432:5432 --name postgres-coordinatord -e POSTGRES_PASSWORD=revault -e POSTGRES_USER=revault -e POSTGRES_DB=coordinator_db postgres:alpine
+docker run --rm -d -p 5432:5432 --name postgres-coordinatord -e POSTGRES_PASSWORD=revault -e POSTGRES_USER=revault -e POSTGRES_DB=postgres postgres:alpine
```
Comment (darosior) [inline CONTRIBUTING.md:58]: Huh? The name of the db is of no importance to the unit tests, they create a `coordinatord_test` DB.

Comment (darosior) [reply CONTRIBUTING.md:58]: Worst case you can just remove the env variable altogether

**Review by darosior — COMMENTED** [review]
LGTM, just needs the revault_net to be merged

File: CONTRIBUTING.md (hunk 1 above)
Comment (danielabrozzoni) [reply CONTRIBUTING.md:58]: It tries to connect to the `postgres` db here:
https://github.com/revault/coordinatord/blob/master/src/processing.rs#L188

If you don't have it it panics...

Comment (danielabrozzoni) [reply CONTRIBUTING.md]: Alright I can't reproduce anymore, it was probably something else. I removed the commit, sorry :)

File: Cargo.lock [hunk 2]
```diff
@@ -691,8 +691,7 @@ dependencies = [
 [[package]]
 name = "revault_net"
 version = "0.1.0"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "7d7581647856ce926493049324bbd0731fb67a34d6ecea676773fd7266617954"
+source = "git+https://github.com/danielabrozzoni/revault_net?branch=20210816_spendtx_null#c6d278f79f390e94e64a59e73d7b76be224e9ace"
```
Comment (darosior) [inline Cargo.lock:694]: Can you update to revault_net master?

**Review by darosior — APPROVED** [review]
ACK 0580cdb63c8ae6fcfbf6599e245a5e7bb860787e

**Pull request merged**